lib/vintage_net_wifi.ex

```diff
@@ -836,4 +841,40 @@ defmodule VintageNetWiFi do
 
     VintageNet.get(["interface", "wlan0", "wifi", "access_points"])
   end
+
+  @doc """
+  Quick way to receive WiFi credentials via WPS.
+  Call this function with a long enough timeout for you to press the WPS button on your access point. As soon as the credentials are received, the function returns.
+  ```elixir
+  VintageNetWiFi.quick_wps(60_000)
+  # Press WPS button on AP
+  {:ok, %{ssid: "MySSID", psk: "MyWiFiPassword"}}
+  ```
+  If you need more control, you can directly initiate `wps_pbc` by calling:
+  ```elixir
+  VintageNet.ioctl("wlan0", :wps_pbc)
+  ```
+  Once credentials (or an error) are recieved they can be read with:
+  ```elixir
+  VintageNet.get(["interface", "wlan0", "wifi", "wps_creds"])
+  ```
```

Comment:
I only made markdown and spelling fixes in the text below.

```suggestion
  Quick way to receive WiFi credentials via WPS
  
  Call this function with a long enough timeout for you to press the WPS button on your access point. As soon as the credentials are received, the function returns.
  
  ```elixir
  VintageNetWiFi.quick_wps(60_000)
  # Press WPS button on AP
  {:ok, %{ssid: "MySSID", psk: "MyWiFiPassword"}}
  ```

  If you need more control, you can directly initiate `:wps_pbc` by calling:

  ```elixir
  VintageNet.ioctl("wlan0", :wps_pbc)
  ```

  Once credentials (or an error) are received they can be read with:

  ```elixir
  VintageNet.get(["interface", "wlan0", "wifi", "wps_creds"])
  ```
```

How do you feel about the "quick" function immediately calling `quick_configure(ssid, password)` to immediately connect to the network.

Reply:
Thought about that, in fact I wrote it like that at first. But then I thought I don't want the function to do more than it has to..
But that's kind of philisophical, since there's probably not much you can do with the credentials other than calling `quick_configure`.
So I'm open to changing it back.

Reply:
My opinion on the `quick_*` functions are that they are only for interactive use at the iex prompt. In that vain, I'd also make a the timeout be the default so that's one less thing to have to type.

I don't think there's a reason to do this, but if a need arises to have a higher level programmatic interface to WPS, then we can add a different function.